src/features/people/PeopleDetails/index.js

```diff
@@ -68,49 +76,49 @@ const PersonDetails = () => {
                         <DetailsTile
                             posterPath={details.profile_path}
                             title={details.name}
-                            description={details.biography}
-                            firstData={details.birthday}
-                            secondData={details.place_of_birth}
+                            description={details.biography || "No biography available"}
+                            firstData={details.birthday || "No date of birth available"}
+                            secondData={details.place_of_birth || "No place of birth available"}
                         />
 
                         <Section>
                             <SectionTitle>Movies - cast ({actorCast.length})</SectionTitle>
-                            <SectionWrapper>
-                                <LargeListWrapper>
-                                    {actorCast.map((cast) => (
-                                        <li key={cast.credit_id}>
+                            <LargeListWrapper>
+                                {actorCast.map((cast) => (
+                                    <li key={cast.credit_id}>
+                                        <StyledLink to={`/movies/${cast.id}`}>
                                             <ListTileLarge
                                                 posterPath={cast.backdrop_path}
                                                 title={cast.title}
-                                                subtitle={getReleaseYear(cast.release_date)}
+                                                subtitle={getReleaseYear(cast.release_date) || "No release year available"}
                                                 tags={nameGenres(cast.genre_ids)}
                                                 voteCount={cast.vote_count}
                                                 ratingValue={formatVote(cast.vote_average)}
                                             />
```

Comment:
I've noticed You do not pass `character={}` prop - we need it in case of movie lists on People Details page.

maybe we should later rename that prop to _role_ or something as we also use in on **crew** tiles

Reply:
Thanks for catching that! 🙌 You're absolutely right. Include the 'character' prop and renaming it to 'role' for consistency sounds like a good plan. We can definitely address that in a subsequent update. Thanks for the feedback 👌